driver-examples/osgi/pom.xml

```diff
@@ -23,27 +23,49 @@
         <artifactId>cassandra-driver-examples-parent</artifactId>
         <version>3.0.1-SNAPSHOT</version>
     </parent>
+
     <artifactId>cassandra-driver-examples-osgi</artifactId>
     <packaging>bundle</packaging>
     <name>DataStax Java Driver for Apache Cassandra Examples - OSGi</name>
     <description>An example of using DataStax Java Driver in an OSGi container.</description>
     <url>https://github.com/datastax/java-driver</url>
 
     <properties>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <felix.version>4.6.0</felix.version>
-        <pax-exam.version>3.6.0</pax-exam.version>
+        <pax-exam.version>4.8.0</pax-exam.version>
```

Comment:
Unfortunately it looks like 4.8.0 requires JDK 7, didn't catch this because our CI bypasses build failures (I should probably look into fixing that).  Switching back to 3.6.0 seems to work fine, do you think we should switch back?